apps/system/js/app_window_manager.js

```diff
@@ -488,6 +490,13 @@
       var activeApp = this._activeApp;
       var detail = evt.detail;
       switch (evt.type) {
+        case 'hierarchytopmostwindowchanged':
```

Comment:
do you think we'll have multiple consumer of this event or should we just listen for `windowopened`/`windowclosed` here?